### PR TITLE
support flask-restplus

### PR DESCRIPTION
**Pull request by yanganto**
I am working on a project with flask and flask-restplus, and I want to integrate with flask-accept.  The swagger document generated by flask-restplus has wrong function description, because of the function doc string is missing.  This issue is fixed, please kindly accept this request.  Thank you.

### CODE REVIEW

File: flask_accept/__init__.py [hunk 1]
```diff
@@ -36,7 +37,12 @@ def __call__(self, *args, **kwargs):
         raise NotAcceptable(description)
 
     def __get__(self, instance, owner):
-        return partial(self.__call__, instance)
+        func = partial(self.__call__, instance)
+
+        # flask-restplus use doc string to swagger docment
+        func.__doc__ = self.__func_doc_str
```
Comment (di) [inline flask_accept/__init__.py]: Could you make this something like:

```python
func.__doc__ = self.fallback.__doc__
```

We don't actually need to store this on the instance. I'm also assuming Swagger can handle the case where the docstring is false-y so we don't need the `if func.__doc__ else ""` but feel free to correct me if that's wrong.

File: flask_accept/test/app_for_testing.py [hunk 2]
```diff
@@ -92,3 +97,44 @@ def get_v2(self):
...
+    @get.support(
+        'application/json',
+        'application/vnd.vendor+json',
+        'application/vnd.vendor.v2+json')
+    def get_v2(self):
+        return jsonify(version='v2')
+
+
+plus_api.add_resource(PlusResourceWithoutFallback, '/plus/without-fallback')
+plus_api.add_resource(PlusResourceWithFallback, '/plus/with-fallback')
```
Comment (di) [inline flask_accept/test/app_for_testing.py]: Would you mind moving these tests to a separate file, something like `test_flask_restplus_integration.py`? (Bonus points if you do something similar with the `flask_restful` tests in this file as well.

File: setup.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
 
-__version__ = '0.0.5'
+__version__ = '0.0.6'
```
Comment (di) [inline setup.py]: Please leave this as-is, I'll bump the version when I make the release.

File: README.rst [hunk 4]
```diff
@@ -217,6 +217,49 @@ The same functionality can be applied to APIs built with Flask-RESTful
     $ curl localhost:5000 -H "Accept: application/vnd.your_vendor.v3"
     Goodbye cruel world.
 
+Works with Flask-RESTPlus Resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The same functionality can be applied to APIs built with Flask-RESTPlus
```
Comment (di) [inline README.rst]: Could you link to https://pypi.org/project/flask-restplus/ here?

**Review by di — COMMENTED** [review]
Thanks @yanganto! Just a few small changes and I think this is good to go.

File: flask_accept/__init__.py (hunk 1 above)
Comment (yanganto) [reply flask_accept/__init__.py]: you are correct :)

**Comment by yanganto** [comment]
@di Ready to merge, thanks for your advice.

**Comment by yanganto** [comment]
@di Ready to merge, thanks for your advice.

File: README.rst (hunk 4 above)
Comment (di) [reply README.rst]: Could you put the link here, on this line, rather than in the title?

File: README.rst [hunk 5]
```diff
@@ -217,6 +217,49 @@ The same functionality can be applied to APIs built with Flask-RESTful
     $ curl localhost:5000 -H "Accept: application/vnd.your_vendor.v3"
     Goodbye cruel world.
 
+Works with `Flask-RESTPlus <https://pypi.org/project/flask-restplus/>`_ Resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The same functionality can be applied to APIs built with Flask-RESTPlus
```
Comment (di) [inline README.rst]: Could you put the link here, on this line, rather than in the title?

File: README.rst [hunk 6]
```diff
@@ -217,6 +217,49 @@ The same functionality can be applied to APIs built with Flask-RESTful
...
+    app = Flask(__name__)
+    api = Api(app)
+
+
+    class HelloWorldResource(Resource):
+        @accept('application/vnd.your_vendor.v1', 'application/vnd.your_vendor.v2')
+        def get():
+        """
+            The doc string showing in swagger
+        """
```
Comment (di) [inline README.rst:239]: Could you indent the docstring quotes here to adhere to [PEP 257](https://www.python.org/dev/peps/pep-0257/)?

**Review by di — COMMENTED** [review]
Two small nitpicks and then I think this is good to go.

File: README.rst (hunk 6 above)
Comment (yanganto) [reply README.rst:239]: Sure :)

File: README.rst (hunk 5 above)
Comment (yanganto) [reply README.rst]: Sure :)

**Review by di — APPROVED** [review]

**Comment by di** [comment]
Thanks @yanganto!

**Comment by yanganto** [comment]
You are welcome.  Thanks for the review.

**Comment by di** [comment]
This has been released: https://pypi.org/project/flask_accept/0.0.6/

**Pull request merged**